runtime/spacetime_byt.c

```diff
@@ -12,8 +12,12 @@
 /*                                                                        */
 /**************************************************************************/
 
+#define CAML_INTERNALS
+
 #include "caml/fail.h"
 #include "caml/mlvalues.h"
+#include "caml/io.h"
```

Comment:
Just out of curiosity, why is `io.h` required here?

Reply:
Because `caml/spacetime.h` uses `struct channel`.  I could have changed `caml/spacetime.h` to include `caml/io.h` itself, but I'm shooting for minimal changes.